changelog.d/pr-7215.md

```diff
@@ -0,0 +1,3 @@
+### 🐛 Bug Fixes
+
+- Reorder commands in ORA get via SSH.  Fixes [#7214](https://github.com/datalad/datalad/issues/7214) via [PR #7215](https://github.com/datalad/datalad/pull/7215) (by [@mslw](https://github.com/mslw))
```

Comment:
```suggestion
- Fix a bug when retrieving several files from a RIA store via SSH, when the annex key does not contain size information.  Fixes [#7214](https://github.com/datalad/datalad/issues/7214) via [PR #7215](https://github.com/datalad/datalad/pull/7215) (by [@mslw](https://github.com/mslw))
```

Reply:
You were too fast, I was editing the changelog offline :/ I like the brevity of yours better, I will make an update.